platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpHandler.java

```diff
@@ -329,7 +329,7 @@ private HandlerResult doPaosRequest(ServletRequest request, ServletResponse resp
             }
             authnRequest = createAndSignAuthnRequest(true,
                     wantSigned && idpssoDescriptor.getWantAuthnRequestsSigned());
-            paosRequest = createPaosRequest();
+            paosRequest = createPaosRequest((HttpServletRequest) request);
```

Comment:
Is it ever possible that the `ServletRequest` would be implemented as something other than `HttpServletRequest` in the future?